tsconfig.json

```diff
@@ -9,6 +9,12 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+    "baseUrl": "./",
+    "paths": {
+      "astarte-client": [
+        "./src/astarte-client/index.ts"
```

Comment:
webpack is in charge of resolving URLs, why do we need to register it here too?

Reply:
This is required so Typescript parsers can resolve the module: i.e. when typescript-eslint parser reads Typescript code